### PR TITLE
Build NuGet packages on AppVeyor

### PR DESCRIPTION
**Pull request by chrarnoldus**

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -1,18 +1,34 @@
...
+  assembly_version: '{version}'
+  file_version: '{version}'
+  informational_version: '{version}'
 environment:
   CONNECTION_STRING: Server=(local)\SQL2016;Database=master;User ID=sa;Password=Password12!
 services: mssql2016
 build_script:
 - ps: >-
-    dotnet build
+    dotnet build -c Release
```
Comment (EamonNerbonne) [inline appveyor.yml]: Is this required given that dotnet pack is called immediately hereafter?

Comment (chrarnoldus) [reply appveyor.yml]: It is required if we want the tools project to be built as well.

File: appveyor.yml [hunk 2]
```diff
@@ -1,18 +1,34 @@
...
   - master
 image: Visual Studio 2017
+dotnet_csproj:
+  patch: true
+  file: src\ProgressOnderwijsUtils\ProgressOnderwijsUtils.csproj
+  version: '{version}'
+  package_version: '{version}'
+  assembly_version: '{version}'
+  file_version: '{version}'
+  informational_version: '{version}'
```
Comment (EamonNerbonne) [inline appveyor.yml]: Do you need any of this other than `version`?

File: appveyor.yml (hunk 1 above)
Comment (EamonNerbonne) [reply appveyor.yml]: So this is both for deploy and testing purposes.

File: appveyor.yml (hunk 2 above)
Comment (chrarnoldus) [reply appveyor.yml]: I don't think so; in fact, AppVeyor only patches the elements that are already present. I just copy-pasted it form the GUI.

**Pull request merged**